components/dashboard/src/projects/NewProject.tsx

```diff
@@ -428,18 +462,54 @@ export default function NewProject() {
         </div>);
     }
 
-    return (<div className="flex flex-col w-96 mt-24 mx-auto items-center">
-        <h1>New Project</h1>
+    const onNewWorkspace = async () => {
+        const redirectToNewWorkspace = () => {
+            // instead of `history.push` we want forcibly to redirect here in order to avoid a following redirect from `/` -> `/projects` (cf. App.tsx)
+            const url = new URL(window.location.toString());
+            url.pathname = "/";
+            url.hash = project?.cloneUrl!;
+            window.location.href = url.toString();
+        }
+        redirectToNewWorkspace();
+    }
+
+    if (!project) {
+        return (<div className="flex flex-col w-96 mt-24 mx-auto items-center">
 
-        {!selectedRepo && renderSelectRepository()}
+            <>
+                <h1>New Project</h1>
 
-        {selectedRepo && !selectedTeamOrUser && renderSelectTeam()}
+                {!selectedRepo && renderSelectRepository()}
 
-        {selectedRepo && selectedTeamOrUser && (<div></div>)}
+                {selectedRepo && !selectedTeamOrUser && renderSelectTeam()}
+
+                {selectedRepo && selectedTeamOrUser && (<div></div>)}
+            </>
+
+            {isBitbucket() && renderBitbucketWarning()}
+
+        </div>);
+    } else {
+        const projectLink = User.is(selectedTeamOrUser) ? `/projects/${project.slug}` : `/t/${selectedTeamOrUser?.slug}/${project.slug}`;
+        const location = User.is(selectedTeamOrUser) ? "" : (<> in <a className="gp-link" href={`/t/${selectedTeamOrUser?.slug}/projects`}>{selectedTeamOrUser?.name}</a></>);
 
-        {isBitbucket() && renderBitbucketWarning()}
+        return (<div className="flex flex-col w-96 mt-24 mx-auto items-center">
 
-    </div>);
+            <>
+                <h1>Project created</h1>
+
+                <p className="mt-2 text-gray-500 text-center text-base">Created <a className="gp-link" href={projectLink}>{project.name}</a>
+                    <br /> {location}
```

Comment:
**Nit**: This deviates slightly from the spec, but I'd find it a lot better to explicitly state what's a "project" and what's a "team", just to avoid any possible confusion here (as we've seen in user testing, users are sometimes confused between workspace vs team vs project, so might as well be 100% clear here).

Also, the new line between the two doesn't make sense, and actually looks like a UI bug:

```suggestion
                <p className="mt-2 text-gray-500 text-center text-base">Created project <a className="gp-link" href={projectLink}>{project.name}</a> {location}
```

Reply:
which spec do you mean? cf. https://gitpod.slack.com/archives/C02EN94AEPL/p1638903077402500?thread_ts=1638877444.381700&cid=C02EN94AEPL

Reply:
To me, spec = visuals added to an issue that show what we want.

Here, it's https://github.com/gitpod-io/gitpod/issues/7098 (see "refined visual" at the bottom of the issue description)

However, I slightly disagree with the spec, because I think:

```
Project Created
Created project [test-project-2] in team [team-1]
```

is clearer than:

```
Project created
Created [test-project-2]
in [team-1]
```